Add documentation on how to contribute to the blog

remixz opened the pull request:
This is based on GH-6, and includes the suggestions from @bnb to keep the process inside of Github, and from @janl about adding guidelines about the Hoodie blog post process. I've also started documenting some of the roles that we may have for our editorial team. 

There's also a rough overview of creating a document for writers about submitting a post in Markdown to this repository, and creating things such as forks and pull requests. Feedback always welcomed! :sparkles:

bnb commented:
I like this a lot. Just a nitpick: I would give the document titles shorter but descriptive names, and I would definitely remove spaces from the file names. For new file names, I would go with something like `pitch-template.md` and `blog-process.md`. 

We'll be setting a precedent that will become hard to change as time goes on, so we need  to get it right the first time. If anyone has a scheme you think might be good, please voice yourself!

remixz commented:
@bnb Thanks for the great feedback! :smile: 

`kebab-case` is definitely nicer for the file names, so I went ahead and renamed the document files in fc7cb16.

I left the title in `CONTRIBUTING.md` long, since I wanted to make it a bit of a generalized document, about any sort of contribution to the editorial team. Of course, that's sort of hard to know when I didn't actually do that in the first commit! :wink: I added a section to `CONTRIBUTING.md` about getting involved with the editorial team in fc7cb16 as well.

For the other titles, I'm open to suggestions. Probably not a huge priority, but it's not hard to change either.

janl commented:
👍👍👍
